src/main/java/com/paneedah/weaponlib/compatibility/CompatibleExtraEntityFlags.java

```diff
@@ -13,13 +13,11 @@
 import net.minecraftforge.common.capabilities.ICapabilitySerializable;
 
 public class CompatibleExtraEntityFlags implements ICapabilitySerializable<NBTBase> {
-    
-    public static int PRONING = 0x01;
+
     public static int FLIP = 0x80000000;
```

Comment:
No idea what `FLIP` is for @Desoroxxx.

Let me know if you feel this is part of the PRONING.

I haven't been able to really understand.

Reply:
Just flags, I've tried changing it to other values, and proning stops working.